client/my-sites/plans/current-plan/purchases-listing.jsx

```diff
@@ -206,7 +206,7 @@ class PurchasesListing extends Component {
 		if ( purchase.autoRenew && ! shouldAddPaymentSourceInsteadOfRenewingNow( purchase ) ) {
 			label = translate( 'Renew now' );
 		}
-		const isProductOwner = Boolean( purchase && purchase.userId === currentUserId );
+		const isProductOwner = Boolean( purchase.userId === currentUserId );
```

Comment:
This change is as suggested in https://github.com/Automattic/wp-calypso/pull/59549/files#r777885084